### ISSUE TITLE
Cannot copy .lnk file to Windows mount

### ISSUE THREAD

**Issue by fquinner** [canonical/multipass#1624]
**Describe the bug**
Since multipass seems to treat `.lnk` files like symlinks, it seems to be impossible to copy actual shortcut files for legitimate reasons (complete with starting directories, comments etc).

**To Reproduce**
* Grab the code from here:
  http://www.mamachine.org/mslink/index.en.html

* Compile it:
  `gcc mslink.c -O2 -o mslink`

* Generate a .lnk file:
  `./mslink -l 'C:\Windows\system32\wscript.exe' -a '"C:\Users\fquinn\.config\wsl-windows-toolbar-launcher\metadata\WSL\silent-launcher.vbs"' -n "About" -o about.lnk`

* Copy this file to the Windows mount

```
ubuntu@primary:~/mslink_v1.3$ cp about.lnk ~/Home/Desktop/
cp: not writing through dangling symlink '/home/ubuntu/Home/Desktop/about.lnk'
```

**Expected behavior**
I expected the `.lnk` to be treated like a regular file. A `.lnk` isn't really the same as a symlink in Linux.

**Logs**
N/A

**Additional info**
 - OS: Windows 10
- `multipass version`: `1.3.0+win`
- `multipass info --all`
```
Name:           primary
State:          Running
IPv4:           172.20.146.127
Release:        Ubuntu 20.04 LTS
Image hash:     d75d900f406d (Ubuntu 20.04 LTS)
Load:           0.00 0.00 0.01
Disk usage:     1.4G out of 4.7G
Memory usage:   516.6M out of 916.9M
Mounts:         C:/Users/fquinn => Home
                    UID map: -2:default
                    GID map: -2:default

Name:           advisable-gar
State:          Running
IPv4:           172.20.155.131
Release:        Ubuntu 20.04 LTS
Image hash:     d75d900f406d (Ubuntu 20.04 LTS)
Load:           0.08 0.02 0.01
Disk usage:     1.3G out of 4.7G
Memory usage:   532.9M out of 916.9M
```

**Additional context**
I run the project https://github.com/cascadium/wsl-windows-toolbar-launcher and I was hoping i could set up GUI menus for multipass too which depend on generating `.lnk` files from the Linux environment hence looking at this. Thanks in advance!

**Comment by townsend2010** [comment]
@fquinner,

Hey, thanks for the bug report.  We'll take a look.

**Comment by townsend2010** [comment]
Hi @fquinner,

I can easily reproduce the behavior you describe and will work on a fix.

In the meantime, there *might* be a workaround depending on your workflow.  The `multipass copy-files` command will copy the `.lnk` file correctly to the host Windows system.  For example, on the host, you would enter something like:
```
$ multipass copy-files primary:path/to/lnk C:\Users\user\Desktop
```
Let us know if that is sufficient for you in the short term.  Thanks!

**Comment by fquinner** [comment]
Yeah I already have a workaround with samba mounting the drive instead which unblocked me until this was fixed but thanks for the other option!

I hit other issues though down the line in #1631 (the idea in this instance is to try and have like a "companion" VM that can act in a way similar to WSL2 but with its own fully virtualized kernel, so I don't want anonymous usernames). I'm sure that's drifting beyond the core goals of Multipass (hence marking that one as an enhancement), but it's how I was trying to use it nevertheless 😄 .

**Comment by tobe2098** [comment]
The current version of multipass supports the copy of Windows shortcuts, and `multipass transfer` covers the same issue. Closing the issue.
